lib/robots/mock/capabilities/MockMapSegmentationCapability.js

```diff
@@ -0,0 +1,32 @@
+const MapSegmentationCapability = require("../../../core/capabilities/MapSegmentationCapability");
+const ValetudoMapSegment = require("../../../entities/core/ValetudoMapSegment");
+
+/**
+ * @extends MapSegmentationCapability<import("../MockRobot")>
+ */
+class MockMapSegmentationCapability extends MapSegmentationCapability {
+    /**
+     * @returns {Promise<Array<import("../../../entities/core/ValetudoMapSegment")>>}
+     */
+    async getSegments() {
+        return [
+            new ValetudoMapSegment({ id: "foo", name: "Foo"}),
+            new ValetudoMapSegment({ id: "boo", name: "Bar"}),
+            new ValetudoMapSegment({ id: "empty" })
```

Comment:
Why does the backend return the "empty" label?

Reply:
I guess I could have written something else. It is just an id for the segment with an empty/undefined name.

Reply:
I changed the IDs to avoid confusion.